doc/configcommands.dsv

```diff
@@ -98,6 +99,7 @@ show-read-feeds|[yes/no]|yes|If yes, then all feeds, including those without unr
 suppress-first-reload|[yes/no]|no|If yes, then the first automatic reload will be suppressed if auto-reload is set to yes.|suppress-first-reload yes
 swap-title-and-hints|[yes/no]|no|If yes, then the title at the top of screen and keymap hints at the bottom of screen will be swapped.|swap-title-and-hints yes
 text-width|<number>|0|If set to a number greater than 0, then all HTML will be rendered to this maximum line length. If set to 0, the terminal width will be used. Does not apply when using external renderer or viewing the source. Also note that "Link" header and "Links" section won't be affected by it—they contain URLs which are better not wrapped.|text-width 72
+toggleitemread-jumps-to-next-unread|[yes/no]|no|If set to yes, jump to the next unread item when an items read status is toggled in the articlelist(Default: N).|toggleitemread-jumps-to-next-unread yes
```

Comment:
1. "an *item's s* status", not "an *items* status".
2. Again, no need to put default value in the description.